File: 최소 신장 트리/9372. 상근이의 여행.py
import sys

def main():
    t = int(input())

    for _ in range(t):
        n, m = map(int, sys.stdin.readline().split())
        for _ in range(m):
            a, b = map(int, sys.stdin.readline().split())
        print(n-1)

if __name__ == "__main__":
    main()

File: BaekJoon/최소 신장 트리/9372_상근이의 여행.swift
let t = Int(readLine()!)!
let inputs = readLine()!.split(separator: " ").map { Int(String($0))! }

for _ 0..<inputs[1] {
    let points = readLine()!.split(separator: " ").map { Int(String($0))! }
    print(points[0] - 1)  
}